draft-ietf-rtcweb-jsep.xml

```diff
@@ -1513,7 +1515,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               new ICE credentials, for the purpose of forcing an ICE
               restart and kicking off a new gathering phase, in which
               the new servers will be used. If the ICE candidate pool
-              has a nonzero size, any existing candidates will be
+              has a nonzero size, and a non-provisional local description
+              has not yet been applied, any existing candidates will be
```

Comment:
Sorry, I don't think I was clear, I am fine with the leftover candidates being unusable. My point is that I think it's unclear when they are discarded. Is it at setLocal() or stable state? Some of the text suggests one and some the other. And if it's at setLocal, why does setConfiguration do anything?